### ISSUE TITLE
Some error I get whenever I start use the launcher after the recent patch

### ISSUE THREAD

**Issue by Simon5Six7** [goaaats/FFXIVQuickLauncher#157]
I have no idea what this is, but I hope this helps in any way

The message:

System.Exception: Game exited prematurely
   at XIVLauncher.Game.XivGame.LaunchGame(String sessionId, Int32 region, Int32 expansionLevel, Boolean isSteamIntegrationEnabled, Boolean isSteamServiceAccount, String additionalArguments, DirectoryInfo gamePath, Boolean isDx11, ClientLanguage language, Boolean closeMutants) in D:\Sapphire\FFXIVQuickLauncher\XIVLauncher\Game\XivGame.cs:line 230
4.6.0.0
cc7e5ba
Context: XG LaunchGame
Microsoft Windows NT 6.2.9200.0
True
True
False
True
True
English

[]

**Comment by Simon5Six7** [comment]
If this is what was meant in the FAQ by "Rich presence doesn't work after update" then I am sorry to waste your time. Just the fact that it crashes at all instead of just a part not working is why I post this at all

**Comment by DAOWAce** [comment]
https://github.com/goaaats/FFXIVQuickLauncher/issues/153

TLDR: It's not going to work until the author updates it for the new patch.

**Comment by Simon5Six7** [comment]
If that's all it is then I apologise for wasting your time :P I thank
